index.html

```diff
@@ -0,0 +1,209 @@
+                <p>Please contact us per Email for any further questions about ICLR Summit 2023! </p>
+                <span><a href="https://iclr.cc/">Kaggle @ICLR 2023></a></span>
+            </div>
+        </div>
+
+    </section>
+    <!-- second part -->
+    <section class="section-prog">
+        <div class="programs">
+            <h2>Main programs
+                <hr>
+            </h2>
+            <!-- list of programs -->
+            <ul class="list-programs">
+                <li class="li-cards">
+                    <img src="images/program_icon_01 1.png" alt="">
+                    <span> Speakers</span>
+                    <p class="p-text">Hear from speakers from diverse nations about the themes of artificial
+                        intelligence and
+                        it's development.
+                    </p>
+                </li>
+                <li class="li-cards">
+                    <img src="images/program_icon_02 1.png" alt="">
+                    <span>Exhibitor</span>
+                    <p class="p-text">
+                        Your donation will be designated to the ICLR DEI Action Fund and will be restricted for use in
+                        providing
+                        accommodations for individuals.
+                    </p>
+                </li>
+
+                <li class="li-cards">
+                    <img src="images/program_icon_03 1.png" alt="">
+                    <span>Forum</span>
+                    <p class="Forum">
+                        Be in a position to spare some time to exchange views and opinions with experts on each topic.
+                    </p>
+                </li>
+                <li class="li-cards">
+                    <img src="images/program_icon_04 1.png" alt="">
+                    <span>Workshop</span>
+                    <p class="workshop">
+                        Endeavor to make your own project with an open source license instead of only watching.
+                    </p>
+                </li>
+                <li class="li-cards">
+                    <img src="images/program_icon_05 1.png" alt="">
+                    <span>Ignite</span>
+                    <p class="ignite">
+                        Acquire opportunities to network with AI developers and affiliates worldwide.
+                    </p>
+                </li>
+            </ul>
+        </div>
+        <button class="footer-but">ICLR 2023 Meeting</button>
+        <h3><a href="#">See the whole program
+                <hr>
+            </a></h3>
+    </section>
+    <!-- third part -->
+    <section>
+        <div id="presenters" class="speakers">
+            <h2>
+                Featured speakers
+            </h2>
+            <!-- dynamically rendered list -->
+
+            <ul class="other-speakers">
+            </ul>
+            <!-- end -->
+            <button type="submit"><span>MORE</span> <i class="fa-solid fa-angle-down"></i></button>
```

Comment:
- [ ] Great job implementing the "MORE" button however, make sure to hide it in the desktop width as it should only appear in the mobile screen width. 
- [ ] Kindly make sure to add an underline to the "Featured Speakers" heading so that it matches the original design. 👍 

Behance Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/227710820-891fff66-c71f-4a58-8117-3edbc57a4d1e.png)  |  ![image](https://user-images.githubusercontent.com/60618877/227710885-ae599f75-d2fd-4930-bc2f-86d461745923.png)